installios.js

```diff
@@ -22,4 +22,4 @@ console.log('Installing pod dependencies');
 execSync('pod update', {stdio:[0,1,2], cwd:'ios'});
 
 console.log('Generating main.jsbundle for tests');
-execSync('react-native bundle --platform ios --dev false --entry-file index.js --bundle-output IntegrationTests/main.jsbundle', {stdio:[0,1,2]});
+execSync('node_modules/.bin/react-native bundle --platform ios --dev false --entry-file index.js --bundle-output IntegrationTests/main.jsbundle', {stdio:[0,1,2]});
```

Comment:
👍 
I think I'm going to get rid of those tests now that we have real tests in the SDK.

Reply:
I didn't see your comment above.
Yeah you can go ahead and clean them up if you want.

Reply:
There is stuff in the podfile https://github.com/forcedotcom/SmartSyncExplorerReactNative/blob/master/ios/Podfile#L41, a couple of test folder, maybe some stuff on the android side as well.